wheatley/bot.py

```diff
@@ -137,7 +140,23 @@ def _on_row_gen_change(self, row_gen_json: JSON) -> None:
 
     def _on_size_change(self) -> None:
         self._check_number_of_bells()
+        self._opening_rounds = generate_starting_row(
```

Comment:
I think it would be clearer if `self._opening_rounds` were renamed to `self._opening_row`, since it isn't necessarily actually rounds.